src/main/blackbox/blackbox.c

```diff
@@ -469,7 +469,8 @@ static bool testBlackboxConditionUncached(FlightLogFieldCondition condition)
     case CONDITION(MOTOR_6_HAS_RPM):
     case CONDITION(MOTOR_7_HAS_RPM):
     case CONDITION(MOTOR_8_HAS_RPM):
-        return (getMotorCount() >= condition - CONDITION(MOTOR_1_HAS_RPM) + 1) && useDshotTelemetry && isFieldEnabled(FIELD_SELECT(RPM));
+        return (getMotorCount() >= condition - CONDITION(MOTOR_1_HAS_RPM) + 1) && useDshotTelemetry
+               && isFieldEnabled(FIELD_SELECT(RPM));
```

Comment:
```suggestion
        return (getMotorCount() >= condition - CONDITION(MOTOR_1_HAS_RPM) + 1) 
               && useDshotTelemetry
               && isFieldEnabled(FIELD_SELECT(RPM));
```
second condition is 'hidden' here. But this one can be easily fixed by hand